edgePy/DGEList.py

```diff
@@ -84,10 +92,10 @@ def __init__(
             self.genes = genes
             self.norm_factors = norm_factors
 
-            if groups_in_dict and groups_in_list:
+            if groups_in_dict is not None and groups_in_list is not None:
```

Comment:
So if someone provides groups_in_dict=False as argument, it is seen as he provided a valid argument there. In which cases is it better to check for None explicitly?

Reply:
Hmm I see it's not so easy..